packages/jellyfish-transaction-builder/__tests__/txn/txn_builder_dex_poolswap.test.ts

```diff
@@ -128,5 +128,96 @@ describe('DFI to DOG', () => {
   })
 })
 
-// TODO(jellyfish): test alternative poolpair
-// TODO(jellyfish): test alternative maxPrice
+describe('DFI to PIG', () => {
```

Comment:
@canonbrother it's a different token pair than the other describe. The first one is `DFI to DOG`.

Reply:
ahhh... my bad...

Reply:
ohh.. got it.. 
btw can you move the `it` into describe('DFI-DOG')?
you should actually refer that fixtures
```diff
-- describe('DFI-DOG') {
  it(should poolswap)
  // yours
  ++ it(should poolswap with max price)
  ++ it(should fail as max price is lower than reserveA/reserveB)
}
```

Reply:
@canonbrother I prefer separating them to avoid flaky tests. Combining both tests for pools causes some inconsistency on the total liquidity and reserves amount.

Reply:
the fixture is set in `beforeEach` `afterEach` so those `it` should be independence, the only changes might affect some test result because of difference liquidity pool set up